README.md

```diff
@@ -0,0 +1,84 @@
+# HashiCorp Vault Add-on for AWS EKS
+
+> Deploy HashiCorp Vault to AWS EKS Clusters with this add-on.
+
+## Table of Contents
+
+- [HashiCorp Vault Addon for AWS EKS](#hashicorp-vault-addon-for-aws-eks)
+  - [Table of Contents](#table-of-contents)
+  - [Usage](#usage)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Usage
+
+If you would like to override any defaults with the chart, you can pass it via the `helm_config` variable.
+
+### Unsealing Vault
+
+Once the add-on has been deployed, the Vault server can be unsealed using the following commands.
+
+> You will need to be in the `vault` (Kubernetes) namespace while running these commands, by default
+
+You will first need to initialize the Vault server:
```

Comment:
```suggestion
First, initialize the Vault server. This will return [unseal keys](https://www.vaultproject.io/docs/concepts/seal#seal-unseal) and [root token](https://www.vaultproject.io/docs/concepts/tokens#root-tokens). Store this in a safe place since you will use them to unseal the Vault server.
```